packages/dashboard/lib/components/FunctionLinks.tsx

```diff
@@ -1,20 +1,22 @@
 import { getCurrentDomain, getFullCurrentDomain, getFullDomain } from 'lib/utils';
 import { Link } from '@lagon/ui';
+import { MouseEventHandler } from 'react';
 
 type FunctionLinksProps = {
   func?: {
     name: string;
     domains: string[];
   };
+  onClick?: MouseEventHandler<HTMLDivElement> | undefined;
```

Comment:
We can remove the `| undefined` at the end, since `onClick` is already optional: 

```suggestion
  onClick?: MouseEventHandler<HTMLDivElement>;
```

Reply:
Ah, nice catch 👍 

Should be fixed now